librz/core/cmd_descs/cmd_debug.yaml

```diff
@@ -19,63 +19,191 @@ commands:
+          - RZ_OUTPUT_MODE_JSON
+          - RZ_OUTPUT_MODE_QUIET
+        args:
+          - name: address
+            type: RZ_CMD_ARG_TYPE_RZNUM
+            optional: true
+          - name: size
+            type: RZ_CMD_ARG_TYPE_RZNUM
+            optional: true
+      - name: dm=
+        cname: cmd_debug_list_maps_ascii
+        summary: List memory maps of current process with ASCII art bars
+        args: []
+      - name: dm.
+        cname: cmd_debug_map_current
+        summary: Show map name of current address
+        args: []
+      - name: dmm
+        summary: Module memory map commands
+        subcommands:
+          - name: dmm
+            summary: List modules of target process
+            cname: cmd_debug_modules
+            modes:
+              - RZ_OUTPUT_MODE_STANDARD
+              - RZ_OUTPUT_MODE_RIZIN
+              - RZ_OUTPUT_MODE_JSON
+            args: []
+          - name: dmm.
+            cname: cmd_debug_current_modules
+            summary: List memory map of current module
+            args: []
+      - name: dm-
+        summary: Deallocate memory map of <address>
+        cname: cmd_debug_deallocate_map
+        args:
+          - name: address
+            type: RZ_CMD_ARG_TYPE_RZNUM
+            optional: false
+      - name: dmd
+        summary: Dump debug map regions to a file (from-to.dmp)
+        subcommands:
+          - name: dmd
+            summary: Dump debug maps to <filename>
+            cname: cmd_debug_dump_maps
+            args:
+              - name: filename
+                type: RZ_CMD_ARG_TYPE_FILE
+                optional: true
+          - name: dmda
+            cname: cmd_debug_dump_maps_all
+            summary: Dump all debug maps
+            args: []
+          - name: dmdw
+            cname: cmd_debug_dump_maps_writable
+            summary: Dump writable debug maps
+            args: []
+      - name: dmh
+        summary: Glibc heap commands
+        subcommands: cmd_heap_glibc
+      - name: dmi
+        summary: List/Load symbols
+        cname: cmd_debug_dmi
+        type: RZ_CMD_DESC_TYPE_OLDINPUT
+      - name: dml
+        summary: Load contents of file into current map region
+        cname: cmd_debug_dml
+        args:
+          - name: file
+            type: RZ_CMD_ARG_TYPE_FILE
+            optional: false
+      - name: dmp
+        summary: >
+          Change page at <address> with <size>, protection <perms>
+          / Change dbg.map permissions to <perm>
+        cname: cmd_debug_dmp
+        args:
+          - name: addr
+            type: RZ_CMD_ARG_TYPE_NUM
+            optional: true
+          - name: size
+            type: RZ_CMD_ARG_TYPE_NUM
+            optional: true
+          - name: perms
+            type: RZ_CMD_ARG_TYPE_STRING
+            optional: false
+      - name: dmL
+        summary: Allocate <size> bytes at <address> and promote to huge page
+        cname: cmd_debug_dmL
+        args:
+          - name: addr
```

Comment:
I would avoid all `addr` arguments, in general, because I think `@ <addr>` modifier is already good enough. So I would make most commands just assume that they have to act on the current address (the user can always use `@ addr` to change it.

Reply:
Should I add "use `@ <addr>` modifier" in the summary of the commands which use that or we just assume the user knows that?

Reply:
Let's just assume the user knows that. I think all commands should assume this behaviour and we should try to make it consistent everywhere, so users know they can use this pattern everywhere.